pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -79,9 +77,17 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
   const _categories =
     layer &&
     _getComponent.data?.categories.map((category: any) => {
-      return categories[layer]?.find((_category) => {
+      const result = categories[layer]?.find((_category) => {
         return _category.value === category;
       });
+
+      if (result === undefined) {
```

Comment:
I think `if (!result)` would do the same. Or if `result === false` or something should also hit the `else`, than maybe not.